projects/collection-editor-library/src/lib/components/question/question.component.ts

```diff
@@ -163,6 +166,22 @@ export class QuestionComponent implements OnInit, AfterViewInit, OnDestroy {
                 }
               }
 
+              if(this.questionInteractionType === 'slider'){
+                if (this.questionMetaData.editorState) {
+                  this.editorState = this.questionMetaData.editorState;
+                  this.sliderOptions=this.questionMetaData.interactions.response1;
+                  this.sliderDatas=this.questionMetaData.interactions.response1;
+                  this.hints =this.questionMetaData.hints;
+                  console.log("editorState");
```

Comment:
always check conditionally. or use lodash to get the values. For ex: 
`this.questionMetaData.interactions.response1` should be fetched as 
`_.get(this.questionMetaData, 'interactions.response1')`